structuredefinitions/UKCore-Observation-Group-Lab.xml

```diff
@@ -1,11 +1,11 @@
 <StructureDefinition xmlns="http://hl7.org/fhir">
     <id value="UKCore-Observation-Group-Lab" />
     <url value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation-Group-Lab" />
-    <version value="2.0.1" />
+    <version value="2.0.2" />
```

Comment:
Should be 2.1.0 as it is a minor update. Patch is used for bug fixes as per semVer

Reply:
can you check now?